edweiss-app/app/(app)/(tabs)/notifications.tsx

```diff
@@ -95,28 +95,13 @@ const NotificationsPage: ApplicationRoute = () => {
 
     // Function to categorize notifications by date
     notifs.forEach((notif) => {
-        const notifDate = new Date(notif.data.date.seconds * timeInMS.SECOND); // Date de la notif
-
-        // Vérifier si la notification est du jour actuel
-        if (notifDate >= midnight) {
-            notifsDay.push(notif);
-        }
-        // Vérifier si la notification est de cette semaine
-        else if (notifDate >= mondayMidnight) {
-            notifsWeek.push(notif);
-        }
-        // Vérifier si la notification est de ce mois-ci
-        else if (notifDate >= monthStart) {
-            notifsMonth.push(notif);
-        }
-        // Vérifier si la notification est de cette année
-        else if (notifDate >= yearStart) {
-            notifsYear.push(notif);
-        }
-        // Si c'est plus ancien, la mettre dans les autres notifications
-        else {
-            otherNotifs.push(notif);
-        }
+        const notifDate = new Date(notif.data.date.seconds * timeInMS.SECOND); // Notif date
```

Comment:
You might want to look at `utils/time.ts`'s `toDate` function instead of doing it by hand here